src/Uno.UI/Controls/NativeCommandBarPresenter.iOS.cs

```diff
@@ -46,11 +75,44 @@ private protected override void OnLoaded()
 			// iOS doesn't automatically update the navigation bar position when the status bar visibility changes.
 			void OnStatusBarChanged(StatusBar sender, object args)
 			{
-				navigationBar.SetNeedsLayout();
-				navigationBar.Superview.SetNeedsLayout();
+				_navigationBar!.SetNeedsLayout();
+				_navigationBar!.Superview.SetNeedsLayout();
 			}
 		}
 
+		CommandBarRenderer RendererFactory() => new CommandBarRenderer(_commandBar?.Target);
+
+		protected override Size MeasureOverride(Size size)
+		{
+			var measuredSize = base.MeasureOverride(size);
+
+			if (_isPhone)
+			{
+				switch (UIDevice.CurrentDevice.Orientation)
+				{
+					// On iPhone, the OS may wrongly report the height of the UINavigationBar:
+					//   - After the creation when it's in landscape mode, it's wrongly
+					//     returning the portrait height as measured height (the width is correct)
+					//   - After the rotation from landscape to portrait, the next measured
+					//     height will be the landscape one. Width is also correct.
+					//
+					// This is a ugly hack to circumvent this annoying OS bug.
+					//
+					// NOTE: those values are been fixed for all iOS versions. Tested on iOS 14.
```

Comment:
I did many tests and search through the doc and it didn't change recently. So we're safe for the past.  My concern is more about the future... since it changed recently on iPad.